deploy/base/config.yaml

```diff
@@ -18,7 +18,6 @@ apiVersion: jvmbuildservice.io/v1alpha1
 kind: UserConfig
 metadata:
   name: jvm-build-config
-  namespace: test-jvm-namespace
```

Comment:
@stuartwdouglas - I think you added this back to my last kcp PR.

For the dev flow, should we not have already switched to the test-jvm-namespace per https://github.com/redhat-appstudio/jvm-build-service/blob/main/deploy/base-development.sh#L13 before this gets applied at https://github.com/redhat-appstudio/jvm-build-service/blob/main/deploy/base-development.sh#L19 ?

Not specifying the namespace here provides some flexibility to use this config in our scenarios besides our classic dev flow like some of the kcp testing I am doing.

Reply:
I was seeing a failure without it, I am not sure why. I did not actually mean to commit it until I had investigated exactly what was going on.

Reply:
ok good to know

I'll try the full openshift dev flow today (vs. the openshift ci flow I usually do) and see if I can reproduced it today.  Maybe it was a minikube only thing?

But let's plan on leaving the namespace setting removed here.  At worst, it is an easy work around for whatever flow needed it.

Reply:
Without it I see:

```
error: no matches for Id jvmbuildservice.io_v1alpha1_UserConfig|test-jvm-namespace|jvm-build-config; failed to find unique target for patch jvmbuildservice.io_v1alpha1_UserConfig|jvm-build-config
```

This is on normal openshift with a new clean namespace.

Reply:
Ah, you just need to delete the namespace from dev-template/config.yaml as well.

Reply:
ok will do that after my apicurio job finishes

thanks